### PR TITLE
Logary and elmah.io example project

### PR DESCRIPTION
**Pull request by ThomasArdal**
Pretty sure this isn't done yet. Had a hard time getting this to compile.

### CODE REVIEW

File: examples/Logary.ElmahIO/Program.fs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    withLogaryManager "Logary.ElmahIO" (
+      withTargets [
+        Console.create Console.empty "console"
+        ElmahIO.create { logId = "GUID_HERE" } "elmah.io"
+      ] >>
+      withRules [
+        Rule.createForTarget "console"
+        Rule.createForTarget "elmah.io"
+      ]
+      withInternalTargets Info [
```
Comment (haf) [inline examples/Logary.ElmahIO/Program.fs]: Let's remove the `withInternalTargets` since it's distracting.

**Comment by haf** [comment]
This is completely correct. Nice.

**Comment by ThomasArdal** [comment]
Removed the withInternalTargets.

File: examples/Logary.ElmahIO/Program.fs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    withLogaryManager "Logary.ElmahIO" (
+      withTargets [
+        Console.create Console.empty "console"
+        ElmahIO.create { logId = "GUID_HERE" } "elmah.io"
+      ] >>
+      withRules [
+        Rule.createForTarget "console"
+        Rule.createForTarget "elmah.io"
+      ]
+      >> run
```
Comment (haf) [inline examples/Logary.ElmahIO/Program.fs]: I think now you have to remove `>> run` since it's internal targets that requires it.

**Comment by haf** [comment]
It's expecting a Guid

```
/home/travis/build/logary/logary/examples/Logary.ElmahIO/Program.fs(29,34): error FS0001: This expression was expected to have type    Guid    but here has type    string
```

**Comment by ThomasArdal** [comment]
Makes sense. Switched.

**Comment by ThomasArdal** [comment]
I don't understand the problem with envForce?

**Comment by ThomasArdal** [comment]
I'll create the c# sample during the weekend.

**Comment by haf** [comment]
Master should work on appveyor now. If you merge master again you'll get CI.

**Comment by ThomasArdal** [comment]
@haf I just fixed a "bug" in paket.references. Rather than referencing the elmah.io package, it should use the elmah.io.client package. This package contains the raw client for communicating with the elmah.io API. The elmah.io package is the implementation for ELMAH, which contains code looking at the HTTP context etc.

**Comment by haf** [comment]
If you push your Builder-error to this PR I can have a look at it later. It's probably an issue with the builder interface for C#; should be quick to fix.

**Comment by ThomasArdal** [comment]
@haf Everything is pushed.

**Comment by haf** [comment]
Rebase on master, too?

**Comment by ThomasArdal** [comment]
@haf Yes, I've synced the fork.

**Comment by haf** [comment]
Hi,

Thanks, I don't have time to look at it the next couple of days. Then I 
can look at it.

Regards,
Henrik

**Comment by ThomasArdal** [comment]
@haf No hurries :smile:

File: paket.lock [hunk 3]
```diff
@@ -115,6 +122,13 @@ NUGET
       Http.fs (>= 3.0.7)
       Logary (>= 4.0.60)
       NodaTime (>= 1.3.2)
+    Logary.Targets.ElmahIO (3.3.1)
```
Comment (haf) [inline paket.lock]: Why add the old target?

**Comment by haf** [comment]
Had some time to look at it. https://www.nuget.org/packages/Logary.Targets.Elmah.Io/ is your package.

**Comment by ThomasArdal** [comment]
@haf I fixed the package name. Cannot see if Appveyor and Travis builds now. Do you know why?

**Comment by haf** [comment]
@ThomasArdal I'm not sure actually. I've tried resyncing and restarting but they both think this is an inactive branch.

**Comment by haf** [comment]
I'm merging for now, so let's try another PR if this one doesn't pan out.

**Pull request merged**